test/shared-class.test.js

```diff
@@ -29,11 +29,6 @@ describe('SharedClass', function() {
       expect(sc.http.path).to.equal('/some');
     });
 
-    it('fills http.path using a normalized path', function() {
-      var sc = new SharedClass('SomeClass', SomeClass, {normalizeHttpPath: true});
```

Comment:
@bajtos regarding preserving backwards compatibility: Is any app constructing `SharedClass` instances directly? The functionality is preserved, just on the level of the `RestAdapter` where it really belongs, and there are unit tests for that... I don't think it makes sense to transform the http routes on the `SharedClass` level, and that's what @ritch's original comment was about, I believe.